scripts/upload_to_gcp.sh

```
@@ -0,0 +1,115 @@
+    fi
+}
+
+return_download_url(){
+    local rdb_download_url=$(gcloud storage sign-url gs://falkordb_rdbs_test_eu/${namespace}/dump.rdb \
+        --duration 12h \
+        --impersonate-service-account falkordb-rdb-storage-reader@pipelines-development-f7a2434f.iam.gserviceaccount.com \
+        --region eu)
+    echo "RDB Download URL: $(echo "$rdb_download_url" | grep signed_url | awk '{print $2}')"
+    
+    if [ "$aof_enabled" = true ]; then
+      local aof_download_url=$(gcloud storage sign-url gs://falkordb_rdbs_test_eu/${namespace}/appendonlydir.tar.gz \
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)